spikeinterface/core/waveform_extractor.py

```diff
@@ -906,6 +955,16 @@ def _reset(self):
         # must be implemented in subclass
         raise NotImplementedError
     
+    def filter_units(self, unit_ids, new_waveforms_folder):
+        # creaste new extension folder
+        new_ext_folder = new_waveforms_folder / self.extension_name
+        new_ext_folder.mkdir()
+        # copy parameter file
+        shutil.copyfile(self.extension_folder / "params.json",
+                        new_ext_folder / "params.json")
+        # specific files must be copied in subclass
```

Comment:
I would excepyt a `self._filter_units(unit_ids, new_waveforms_folder)` after the commoin part
and then `def __filter_units()` that must  overwrite in subclass. No ?

Reply:
ok